next/components/forms/Button.tsx

```diff
@@ -188,31 +218,53 @@ const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, PolymorphicProp
           className={style}
           {...buttonPropsFixed}
         >
-          <i className={size === 'sm' ? 'ml-2' : 'ml-3'}>
-            <ArrowRightIcon
-              width={size === 'sm' ? LINK_ICON_SM : LINK_ICON_LG}
-              height={size === 'sm' ? LINK_ICON_SM : LINK_ICON_LG}
-            />
+          <i
+            className={`flex justify-center items-center ${size === 'lg' ? 'ml-2 h-6 w-6' : ''} ${
+              size === 'sm' ? 'w-5 h-5 ml-1' : ''
+            }`}
+          >
+            <ArrowRightIcon />
           </i>
         </MLink>
       )
     }
 
     return (
-      <button type="button" ref={ref as RefObject<HTMLButtonElement>} className={style} {...buttonProps}>
+      <button
+        type="button"
+        ref={ref as RefObject<HTMLButtonElement>}
+        className={style}
+        {...buttonProps}
+      >
         <div className="flex items-center">
           {startIcon && (
-            <i className={`${size === 'lg' && 'mr-3 h-5 w-5'} ${size === 'sm' && 'mr-2.5 h-4 w-4'}`}>{startIcon}</i>
+            <i
```

Comment:
Why do we use `<i>` element? It stands for "idiomatic text", not for "icon" :) 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i